.evergreen/generate_evergreen_tasks.js

```diff
@@ -433,7 +448,7 @@ for (const {
     const nodeLtsDisplayName = `Node${NODE_LTS_VERSION}`;
     const name = `${osName}-${NODE_LTS_VERSION >= 20 ? nodeLtsDisplayName : nodeLTSCodeName}`;
     const display_name = `${osDisplayName} ${nodeLtsDisplayName}`;
-    const expansions = { NODE_LTS_VERSION };
+    const expansions = { NODE_LTS_VERSION, NPM_VERSION: NODE_LTS_VERSION === 16 ? 9 : 'latest' };
```

Comment:
```suggestion
    const expansions = { NODE_LTS_VERSION, NPM_VERSION: NODE_LTS_VERSION <= 16 ? 9 : 'latest' };
```

Node 16 is fixed but we still test node 14 on 5.x, we're going to run into the same issue on the 4.x branch with node 12.